### ISSUE TITLE
Atom not opening window

### ISSUE THREAD

**Issue by ryrichard** [atom/atom#22908]
### Prerequisites

* [x ] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description

I have a Windows 7. I cannot open atom anymore. It use to work a couple of months ago, but when I tried opening it now, no window pops up. I can see it is open in the task manager, but no window opens. I tried the command prompts to run safe mode, but atom was not recognized.  added the path in the system variables. The error does not appear anymore, now it just says nothing. 

### Steps to Reproduce

1. Double click atom

**Expected behavior:**

1. Double click atom
2. atom opens

**Actual behavior:**

1. Double click atom
2. nothing happens

**Reproduces how often:**

Seems 100% now. I tried a few days ago. Thought that a computer reset would fix it. Tried it today and the problem persist

### Versions

Atom 1.5.8
OS: Window 7

**Comment by wlwl2** [comment]
Do you want the latest updates to work on Windows 7? Would it be alright if you used an older version of atom?

**Comment by ryrichard** [comment]
yes, that would be great. thank you. Does this mean that future updates for atom no longer support windows 7?

**Comment by wlwl2** [comment]
> Does this mean that future updates for atom no longer support windows 7?

No, it was to get an understanding of the situation, so updates in the future are inclusive.

**Comment by wlwl2** [comment]
Try this: Uninstall your current version and then go to: https://github.com/atom/atom/tags. Try out some older releases by clicking on their Downloads button/link. When atom is installed uncheck the automatically download updates in About. Note: I don't guarantee this can work, it's just a suggestion.
